lib/ores_api.rb

```diff
@@ -57,8 +66,31 @@ def query_url(rev_ids)
   end
 
   def ores_server
-    conn = Faraday.new(url: 'https://ores.wikimedia.org')
+    conn = Faraday.new(url: ORES_SERVER_URL)
     conn.headers['User-Agent'] = ENV['dashboard_url'] + ' ' + Rails.env
     conn
   end
+
+  def report_exception(error, url, level: 'error', sentry_tag_uuid: nil)
```

Comment:
Why is all this error-handling code in OresApi instead of the error-handling module?

Reply:
I refactored the error handling code in the three APIs to `ErrorHandling` module.